### PR TITLE
Add multiselect support to Combobox

### PR DESCRIPTION
**Pull request by osamasayed**
### Summary
This PR adds multi-select support to the current Combobox component.

### Screenshots

| Before | After |
| ------ | ------ |
||Screen Shot 2021-08-29 at 18 42 52|
||Screen Shot 2021-08-29 at 18 41 42|
||Screen Shot 2021-08-29 at 18 42 22|

### CODE REVIEW

File: src/components/dls/Forms/Combobox/ComboboxItem.tsx [hunk 1]
```diff
@@ -47,23 +47,24 @@ const ComboboxItem: React.FC<Props> = ({
           styles.itemContainer,
           { [styles.disabledItem]: disabled },
           { [styles.enabledItem]: !disabled },
+          { [styles.checkedItem]: checked },
         )}
       >
         <input
-          type="radio"
+          type="checkbox"
```
Comment (tifa2UP) [inline src/components/dls/Forms/Combobox/ComboboxItem.tsx:54]: should this be conditional radio/conditional based on the multiselect prop?

File: src/hooks/useFocusElement.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { useRef, RefObject } from 'react';
+
+/**
+ * A hook that focus a referenced element.
+ *
+ * @param {FocusOptions} options
+ * @returns {[() => void, RefObject<T>]}
+ */
+export const useScrollToElement = <T extends HTMLElement>(
```
Comment (tifa2UP) [inline src/hooks/useFocusElement.ts:9]: Is the function used to scroll to Element to focus an element? 

nit: focuses*

File: src/components/dls/Forms/Combobox/ComboboxItem.tsx (hunk 1 above)
Comment (osamasayed) [reply src/components/dls/Forms/Combobox/ComboboxItem.tsx:54]: This is done intentionally as before, there was an issue even with single select which is once you select you cannot un-select due it being a radio and not a checkbox.

File: src/hooks/useFocusElement.ts (hunk 2 above)
Comment (osamasayed) [reply src/hooks/useFocusElement.ts:9]: My bad. This needs renaming

**Review by tifa2UP — APPROVED** [review]

**Pull request merged**